requirements.txt

```diff
@@ -29,9 +29,9 @@ tqdm==4.41.1
 word2number==1.1
 transformers==3.3.1
 https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz
-coverage
-flask-cors
-flask
+coverage==5.3
+flask-cors==3.0.9
```

Comment:
exact versions seem an overkill.

I prefer if you change it to something like `flask-cors>=3.0`

Reply:
I was worried about package backwards compatibility issue but realized that we can rely on CI to detect it. So aggressively changed all package version req to `>=` except for `transformer` which requires a certain version.